### PR TITLE
SP-3 Base setup for testing

### PR DESCRIPTION
**Pull request by kancha-supriya**
Please review

### CODE REVIEW

File: tests/testutils/db.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_product_live_time_table():
+    """Create product_live_time table."""
+    with rds_db_session() as session:
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: you dont need to do these things. directly put base_model.metadata.create_all(rds_db_engine)

File: tests/testutils/db.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        _exit_if_not_test_environment(session)
+        base_model.metadata.create_all(rds_db_engine)
+
+
+def drop_product_live_time_table():
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: change the function name, make it related to timed_release model class name

File: tests/testutils/db.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_timed_release_model_table():
+    """Create Model table."""
+    _exit_if_not_test_environment(rds_db_session())
+    base_model.metadata.create_all(rds_db_engine)
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: This create_all creates a table for all the model classes where base_model has been injected.

File: tests/testutils/db.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+from functools import wraps
+import sys
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_all_table():
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: create_all_tables() - plural

File: tests/testutils/db.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    """Create a database table for the respective model classes where.
+
+    base_model is inherited.
+    """
+    _exit_if_not_test_environment(rds_db_session())
+    base_model.metadata.create_all(rds_db_engine)
+
+
+def drop_all_table():
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: plural

File: tests/testutils/db.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_all_table():
+    """Create a database table for the respective model classes where.
+
```
Comment (pravinbattulasts) [inline tests/testutils/db.py]: can we do something for this blank line

Comment (kancha-supriya) [reply tests/testutils/db.py]: no its gives error while running flake 8 
either i have to shorten the sentence or let it be as it is

**Comment by pravinbattulasts** [comment]
A+

**Pull request merged**